estate/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Real Estate",
+    "author": "nmak",
+    "website": "https://www.odoo.com/apps/estate",
+    "category": "tutorials/estate",
+    "depends": ["base"],
+    "data": [
+        "security/ir.model.access.csv",
+        "views/estate_property_views.xml",
+        "views/estate_property_type_views.xml",
+        "views/estate_property_tag_views.xml",
+        "views/estate_menus.xml",
+    ],
+    "installable": True,
+    "application": True,
+    "auto_install": False,
+    "license": "LGPL-3",
```

Comment:
Smae
```suggestion
    'license': 'LGPL-3',
```
Same as above